packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -451,6 +451,15 @@ class FlutterDriver {
     await _sendCommand(new EnterText(text, timeout: timeout));
   }
 
+  /// If `enabled` is true, enables text entry emulation via [enterText]. If
```

Comment:
This could provide some more details about the difference between emulation enabled and disabled. E.g. when should you disable it?

Reply:
Good point: https://github.com/flutter/flutter/pull/13420